### ISSUE TITLE
[bslint] Create a bslint.json jsonschema

### ISSUE THREAD

**Issue by bartvandenende-wm** [rokucommunity/bslint#104]
## Summary

Create a bslint.json jsonschema

## Details

it would be great if bslint.json would have a json schema to improve IDE support (ex. on inline-docs, autocompletion and warnings), for example
```json
{
   "$schema": "https://{som-url}/bslint.schema.json",
  "rules": {},
  "globals": [],
  "ignores": []
}
```

**Comment by TwitchBronBron** [comment]
Yeah, this would be great! Pretty sure schemas can be referenced locally too, so it could be bundled with bslint and referenced like:

```json
{
   "$schema": "./node_modules/@rokucommunity/bslint/schema.json",
  "rules": {},
  "globals": [],
  "ignores": []
}
```

But I'd also be open to embedding it into the [vscode extension](https://github.com/rokucommunity/vscode-brightscript-language) so all roku devs get it automatically.

Any chance you'd be interested in working on this one?
